front-end/src/pages/SettingsPage.tsx

```diff
@@ -35,9 +35,9 @@ const SettingsPage = (props: Props) => {
 
     // get current author data
     function getAuthorData() {
-        AxiosWrapper.get(authorUrl).then(res => {
+        AxiosWrapper.get(authorUrl).then((res: any) => {
             setUnchangedData(res.data);
-        }).catch(err => {
+        }).catch((err: any) => {
```

Comment:
If anyone is curious about what can be done to type this down the road, this might be useful: https://stackoverflow.com/questions/62217642/react-and-typescript-which-types-for-axios-response

The current implementation is good as is as it localizes `any` to a few method calls, and we didn't really use exhastive typing on `res` or `err` to begin with :+1:

Reply:
Thanks for this! This clears things up.